client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -116,9 +120,23 @@ const Header: FunctionComponent = () => {
 
 	const handleSignup = () => {
 		setShowSignupDialog( true );
+		setShowLoginDialog( false );
+	};
+
+	const handleLogin = () => {
+		setShowSignupDialog( false );
+		setShowLoginDialog( true );
+	};
+
+	const closeAuthDialog = () => {
+		setShowSignupDialog( false );
+		setShowLoginDialog( false );
 	};
 
 	useEffect( () => {
+		// make login a part of this
+		// we need to add something to determine if the user exists I think, and then just redirect.
```

Comment:
Would `select( USER_STORE ).getCurrentUser()` do the trick?

Reply:
oops those comments were obsolete!